lib/node/runner/api/api.go

```diff
@@ -56,6 +60,50 @@ func (api NetworkHandlerAPI) HandlerURLPattern(pattern string) string {
 	return fmt.Sprintf("%s/%s%s", api.urlPrefix, api.version, pattern)
 }
 
+func TriggerEvent(st *storage.LevelDBBackend, transactions []*transaction.Transaction) {
+	accountMap := make(map[string]struct{})
+	for _, tx := range transactions {
+		source := tx.Source()
+		accountMap[source] = struct{}{}
+		txHash := tx.H.Hash
+
+		txEvent := observer.NewSubscribe(observer.NewEvent(observer.ResourceTransaction, observer.ConditionAll, "")).String()
```

Comment:
IHMO,  `txEvent` is kind of event. but  it has events. 
What do you think that like below:
```
type Events []Event
func (es Events) String() string {
     var sb []string
     for _, e := range es {
         sb = append(sb,e)
    }
    return strings.Join(sb," ")
}

txEvent := o.Events{
    o.NewEvent(o.ResourceTransaction,o.ConditionAll,""),
    o.NewEvent(o.ResourceTransaction,o.ConditionSource,source),
}

go observer.ResourceObserver.Trigger(txEvent.String(), &opBlock)

```

Reply:
oh it is better.

Reply:
But,
I want to make an `and` condition
As you can see here,
```
type Subscribe struct {
	Events []Event `json:"resources"`
}
func (s Subscribe) String() string {
	toStr := ""
	for i, e := range s.Events {
		toStr += e.String()
		if i == len(s.Events)-1 && len(s.Events) != 1 {
			toStr += "&"
		}
	}
	return toStr
}
```

Events in Subscribe are combined with `AND` condition.

to make `OR` condition, []Subscribe is needed.

Reply:
What means of `&'? :-> We can use it like `o.Trigger(o.Subscribe(events...)..)`?

Reply:
yes ~ we can use the trigger like that

subscribes are 'or' condition
events are 'and' condition

Reply:
event things are refactored. please review again~